src/Collection/ItemPriceCollection.php

```diff
@@ -66,13 +66,14 @@ public function count()
     /**
      * Retrieves the total price of all items within the collection including taxes without discounts
      *
+     * @param string $tax_type Type of taxes to include (optional)
      * @return float The total price including taxes without including discounts
      */
-    public function totalAfterTax()
+    public function totalAfterTax($tax_type = null)
```

Comment:
As an aside. It also might be a good time to introduce class constants to hold tax types

```php
<?php
class TaxPrice extends AbstractPriceModifier
{
    const INCLUSIVE = 'inclusive';
    const EXCLUSIVE = 'exclusive';

```

That way you can use `TaxPrice.INCLUSIVE` instead of `'inclusive'`.

Reply:
What is the advantage of using constants like this?

Reply:
You eliminate typos, which would be found at compile time/build time (via linting)